scripts/openstack-fake-update.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+#
+# Copyright 2024 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+set -ex
+OUTFILE=${OUTFILE:-csv.yaml}
+TIMEOUT=${TIMEOUT:-300s}
+VERSION=${VERSION:-0.0.2}
+
+if [ -z "$OPERATOR_NAMESPACE" ]; then
+    echo "Please set OPERATOR_NAMESPACE"; exit 1
+fi
+if [ -z "$NAMESPACE" ]; then
+    echo "Please set NAMESPACE"; exit 1
+fi
+
+OPENSTACK_OPERATOR_CSV=$(oc get csv -n $OPERATOR_NAMESPACE -o name | grep openstack-operator)
```

Comment:
This is forcing a fake update by adding another entry with the same SHAs. This is a good option to keep, but make room to include a real index with two bundles and executing the test against that if such a thing is present.

Reply:
with new param fake_update there is a room to use new real index